releng/release-tool/release-tool_test.go

```diff
@@ -457,3 +460,37 @@ func TestNewTag(t *testing.T) {
 		}
 	}
 }
+
+func Test_convertRunBeforeMergeToAlwaysRun(t *testing.T) {
```

Comment:
nit: Without the _

Reply:
As said in another PR, since this is a non-exported function the IDE would not find it. Also it's more clear and less annoying to read (IMHO) - but if you insist...

Reply:
![image](https://github.com/user-attachments/assets/fdc4fd1b-dbe1-4062-8b2b-08bd0969bce3)

^^ this is what I mean